public/robots.txt

```diff
@@ -1,2 +1,2 @@
 User-agent: *
-Disallow:
+Disallow: /
```

Comment:
I think we could probably just remove this file since it's in our `.gitignore`. Since this file gets created during the build though, it should be okay to leave it in like this

Otherwise if you can, could we do a `git rm` of it?